lib/linter-glsl.js

```diff
@@ -57,16 +61,36 @@ const parseGlslValidatorResponse = (inputs, output) => new Promise((resolve) =>
     let compileStarted = false;
     const typeName = shader.type.name;
 
+    let consumer;
+    if (shader.sourcemaps) {
+      consumer = new sourceMap.SourceMapConsumer(shader.sourcemaps);
+    }
+
     output.split(os.EOL).forEach((line) => {
       if (line.endsWith(shader.name)) {
         compileStarted = true;
       } else if (compileStarted || inputs.length === 1) {
         const match = new RegExp(compileRegex).exec(line);
         if (match) {
-          const lineStart = parseInt(match[3], 10);
-          const colStart = parseInt(match[2], 10);
-          const lineEnd = lineStart;
-          const colEnd = colStart;
+          let l = parseInt(match[3], 10);
```

Comment:
Just use `lineStart` and `colStart` as the original code, there is no reason to switch to obscure names only to put them back to understandable ones right after.